ChangeLog.rst

```diff
@@ -6,6 +6,7 @@ Note worthy changes
 
 - The ``instagram`` provider now extracts the user's full name.
 - New provider: NextCloud (OAuth2)
+- Stripe Connect Express provider added. API more or less the same as Stripe provider with an addtional option or two
 
```

Comment:
Can you please make sure to check the checklist for adding a new provider?:
 In case you add a new provider:
 
- [ ] Make sure unit tests are available.
- [ ] Add an entry of your provider in `test_settings.py::INSTALLED_APPS` and `docs/installation.rst::INSTALLED_APPS`.
- [ ] Add documentation to `docs/providers.rst`.
- [ ] Add an entry to the list of supported providers over at `docs/overview.rst`.